src/main/java/com/fewlaps/quitnowemailsuggester/EmailSuggester.java

```diff
@@ -114,7 +114,7 @@ private String fixDomain(String email, String badDomain, String goodDomain) thro
         return email;
     }
 
-    private String fixDomainAndTld(String email, String badDomainAndTld, String goodDomainAndTld) {
+    private static String fixDomainAndTld(String email, String badDomainAndTld, String goodDomainAndTld) {
```

Comment:
Hi @rocboronat this link may be helpful https://github.com/jayway/powermock/wiki/MockitoUsage#a-full-example-for-mocking-stubbing--verifying-static-method